common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -434,6 +434,18 @@ def validate(self):
         validation = super(LibraryContentDescriptor, self).validate()
         if not isinstance(validation, StudioValidation):
             validation = StudioValidation.copy(validation)
+        library_tools = self.runtime.service(self, "library_tools")
+        if not (library_tools and library_tools.can_use_library_content(self)):
+            validation.set_summary(
+                StudioValidationMessage(
+                    StudioValidationMessage.ERROR,
+                    _(
+                        u"This course/system does not support content libraries. You probably "
```

Comment:
Do we need to mention "system"? @ kelketek @marco @antoviaque is this message still accurate if we say "This course does not support content libraries."?
I would also prefer stating something like "Only courses that are created using the split modulestore support content libraries", rather than suggesting that they create new courses using the split modulestore. Your thoughts?